gluon/html.py

```diff
@@ -2587,16 +2619,16 @@ def xml(self):
             return self.serialize(self.data, 0).xml()
 
 
-def embed64(
-    filename=None,
-    file=None,
-    data=None,
-    extension='image/gif',
-):
+def embed64(filename=None,
+            file=None,
+            data=None,
+            extension='image/gif'
+            ):
     """
     helper to encode the provided (binary) data into base64.
 
     Args:
+        extension:
```

Comment:
If you're making a point on rearraging the code this should come AFTER every other argument :P